Source/Core/DolphinWX/ISOProperties/ISOProperties.cpp

```diff
@@ -231,13 +231,16 @@ void CISOProperties::CreateGUIControls()
 {
   const int space5 = FromDIP(5);
 
-  wxButton* const edit_config = new wxButton(this, ID_EDITCONFIG, _("Edit Config"));
-  edit_config->SetToolTip(_("This will let you manually edit the INI config file."));
+  wxButton* const edit_config = new wxButton(this, ID_EDITCONFIG, _("Edit User Config"));
+  edit_config->SetToolTip(
+      _("This will let you edit the user config INI file for this game. Changes should be "
```

Comment:
I know the old writing had a lot of this, but if we're touching it, let's make it better! First up, "you" is not great writing and in general should be avoided if possible. And Config should probably not be shortened in the tooltip. Also I think stating that editing should be here is a little redundant when it says not to do it on the other.

I recommend: "Allows manual editing of the user configuration INI file for this game."